_coverpage.md

```diff
@@ -1,8 +1,10 @@
-<p align="center">
-  <a href="https://flow-launcher.github.io">
-	<img width="250px" src="https://raw.githubusercontent.com/Flow-Launcher/Flow.Launcher/5ba4514f31e624c679628d4dfe89036c0e24006c/Doc/Logo/resources/flow-header-square-transparent.png">
-  </a>
-</p>
+<a href="https://www.flowlauncher.com/">
+    <img
```

Comment:
personally I don't really like this style of formatting. Don't you think ending with a single `>` is awkward 🤣? Though if that's common it's fine.

Reply:
I know some people prefer it like this: 
```html
<img
  width="250"
  alt="description"
  src="https://somelongurl">
```
I personally find it extremely confusing because I see the opening `<` but don't see the closing `>` in the same column. It's the same reason a lot of people put `{` and `}` in C# in the same column: 
```cs
if(condition)
{
  // stuff
}
```

even though in many other languages people do this:
 ```js
if (condition) {
  // stuff
}
```

I could just leave it on one long line if you prefer.

Reply:
A few open-source examples:

* [PrimeVue](https://github.com/primefaces/primevue/blob/d7c25df7b02a5c9791be50a7769201b3508c0d59/components/lib/accordion/Accordion.vue#L4-L10)
* [Vuetify](https://github.com/vuetifyjs/vuetify/blob/3175d9b9bc6b3742faa2c54110ba758ab12f40b7/packages/vuetify/src/components/VAutocomplete/VAutocomplete.tsx#L419-L446)
* [Ionic](https://github.com/ionic-team/ionic-framework/blob/84f5defe33f57352076a8cfb2a139705597f7a21/core/src/components/button/button.tsx#L347-L384)

Reply:
I think it's that `>` is too small compared to `}` which occupy a whole column. Anyway if that's common then it's fine.